MODULE.bazel

```diff
@@ -0,0 +1,26 @@
+module(
+    name = "rules_angular",
+    version = "0.0.0",
+)
+
+bazel_dep(name = "aspect_bazel_lib", version = "2.14.0")
+bazel_dep(name = "aspect_rules_js", version = "2.3.3")
+bazel_dep(name = "aspect_rules_ts", version = "3.5.1")
+
+npm = use_extension("@aspect_rules_js//npm:extensions.bzl", "npm")
+npm.npm_translate_lock(
+    name = "rules_angular_npm",
+    npmrc = ":.npmrc",
+    data = [
+        ":package.json",
+        ":patches/@angular__compiler-cli.patch",
+    ],
+    pnpm_lock = ":pnpm-lock.yaml",
+)
+use_repo(npm, "rules_angular_npm")
+
+rules_ts_ext = use_extension("@aspect_rules_ts//ts:extensions.bzl", "ext")
```

Comment:
I haven't picked up bzlmod yet, but yesterday I've made the Angular compiler version configurable; this probably is outdated now?

Reply:
Outdated in what sense? As long as you have typescript sources in this repo that end-users need to build (as opposed to checking in "dist/thing.js") then this will still need to be here

Reply:
I meant that there is another `setup_3` step now, that requires users to pass the Angular compiler-cli and typescript version.

Building from source is what we want, yeah.

Reply:
Yes I've been studing the step3 - the right answer :tm: is that users should provide deps that they install, it's never feasible for a starlark module to also be a proper npm dependency.

Reply:
For now this wasn't even needed, so I reduced the PR to the minimum to run architect